samples/coop_synchronization/sample.yaml

```diff
@@ -0,0 +1,37 @@
+sample:
+  description: A simple application that demonstrates even more basic sanity of the kernel
+    compared to samples/synchronization. The main variant is the polled_uart which shouldn't
+    go through any interrupt driven context switch in a no-systick scenario.
+  name: Coop Synchronization Sample
+tests:
+  sample.kernel.coop_synchronization.polled_uart:
+    build_on_all: true
+    # Platforms below do not implement a polled uart driver therefore they dont build
+    platform_exclude: rpi_pico adafruit_kb2040 m2gl025_miv s32z270dc2_rtu0_r52 s32z270dc2_rtu1_r52
+      sparkfun_pro_micro_rp2040
+    tags: synchronization
+    extra_configs:
+      - CONFIG_UART_INTERRUPT_DRIVEN=n
+    harness: console
+    harness_config:
+      type: multi_line
+      regex:
+        - "thread_a: level 1 irq_lock key = (\\d+)"
+        - "thread_a: level 2 irq_lock key = (\\d+)"
+        - "thread_a: Hello World from cpu (\\d+) on (\\w+)! counter = (\\d+)"
+        - "thread_b: level 1 irq_lock key = (\\d+)"
+        - "thread_b: level 2 irq_lock key = (\\d+)"
+        - "thread_b: Hello World from cpu (\\d+) on (\\w+)! counter = -(\\d+)"
+  sample.kernel.coop_synchronization.interrupt_driven_uart:
+    build_on_all: true
```

Comment:
please remove this one, this is reserved for limited usage in CI.

Reply:
In both requests, you mean specifically and only the 
```   
    build_on_all: true
```
line right?